tests/repository_data/targets_simpler.json

```diff
@@ -0,0 +1,54 @@
+{
```

Comment:
(Do you think it would be better to just stick them in with the 'repository' stuff?)